### PR TITLE
chore(litmus-portal): Added loader in verify and commit finish button

### PR DESCRIPTION
**Pull request by amityt**
Signed-off-by: Amit Kumar Das <amitkumar.das@mayadata.io>

## Proposed changes

This PR adds:
- Loader in the finish button of verify and commit screen

Fixes the issue #2549 

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

## Special notes for your reviewer:
Screenshot from 2021-03-16 11-32-30

### CODE REVIEW

File: litmus-portal/frontend/src/components/WorkflowStepper/index.tsx [hunk 1]
```diff
@@ -477,11 +478,15 @@ const CustomStepper = () => {
               </ButtonOutline>
               {activeStep === steps.length - 1 ? (
                 <ButtonFilled
-                  isDisabled={validateWorkflowName(name)}
+                  isDisabled={validateWorkflowName(name) || loading}
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/WorkflowStepper/index.tsx:481]: can't we use the buttons from litmus-ui or do you want to tackle all that in a separate PR?

**Review by arkajyotiMukherjee — APPROVED** [review]

File: litmus-portal/frontend/src/components/WorkflowStepper/index.tsx (hunk 1 above)
Comment (amityt) [reply litmus-portal/frontend/src/components/WorkflowStepper/index.tsx:481]: I'm updating in the same PR. Thanks for pointing this out. @arkajyotiMukherjee

Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/components/WorkflowStepper/index.tsx:481]: The only thing is... it's used in a lot of places...do you want to do all that in this PR?

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Review by S-ayanide — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]

**Pull request merged**